src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -70,6 +83,19 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * java .lang .Integer foo = 1;
+ *  // ^     ^ violation, No whitespace before '.' is allowed
```

Comment:
1. no need for 2 examples, it can be `Map. Entry e = null; // violation comment`
2. `No` to lower case